charts/zeebe-benchmark/values.yaml

```diff
@@ -223,6 +224,8 @@ camunda-platform:
         value: "0"
       - name: PYROSCOPE_LABELS
         value: "namespace=$(K8S_NAMESPACE), pod=$(K8S_NAME)"
+      - name: SPRING_CONFIG_ADDITIONALLOCATION
```

Comment:
:clown_face: I keep reading this as Addition Allocation :smile:

Reply:
Oh my, I was wondering what the hell spring wants to allocate until I read your comment :facepalm:

Reply:
Thanks for the input you both :) I added an additional comment to clarify this a bit.